example/deploy-originated-integration-tests-contracts.ts

```diff
@@ -132,7 +136,12 @@ async function checkBalances(users: string | any[]) {
 async function originateTheContracts() {
   await checkBalances(users);
 
-  console.log('originating...');
+  console.log('originating...')
+  contract_catalogue.set('KnownContract', await originateKnownContract());
+  contract_catalogue.set('knownBigMapContract', await originateknownBigMapContract());
+  contract_catalogue.set('SaplingStateContract', await originateSaplingStateContract());
+  contract_catalogue.set('Tzip12BigMapOffChain', await originateTzip12BigMapOffChain());
```

Comment:
Are these contracts used for the examples on the website?

Reply:
I thought this file was use to deploy the contracts used on the website. Why is it renamed to example/deploy-originated-integration-tests-contracts.ts?

Reply:
I have changed the name again to:  deploy-contracts-for-website-and-integration-tests.ts

Reply:
i will add indications that separate thecontracts for the integration tests from the contracts for the website.

Reply:
```
 //Contracts used in integration tests. Use these in ~/taquito/integration-tests/config.ts
  console.log('Originating contracts used in integration tests...')
  contract_catalogue.set('KnownContract', await originateKnownContract());
  contract_catalogue.set('knownBigMapContract', await originateknownBigMapContract());
  contract_catalogue.set('SaplingStateContract', await originateSaplingStateContract());
  contract_catalogue.set('Tzip12BigMapOffChain', await originateTzip12BigMapOffChain());

  //Contracts used in live code examples in the Taquito Docs Website
  console.log('Originating contracts used in the Taquito docs Website...')
  contract_catalogue.set('IncrementContract', await originateIncrementContract());
  contract_catalogue.set('MichelsonMapContract', await originateMichelsonMap());
  contract_catalogue.set('LambdaViewContract', await originateLambda1());
```